config/galaxy.ini.sample

```diff
@@ -633,6 +638,29 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy also manages the docker containers more closely than the
+# NodeJS proxy, so is able to expose more container management related options
+
+# This attribute governs the minimum length of time between consecutive HTTP/WS
+# requests through the proxy, before the proxy considers a container as being
+# inactive and kills it.
+#dynamic_proxy_golang_noaccess = 60
+
+# In order to kill containers, the golang proxy has to check at some interval
+# for possibly dead containers. This is exposed as a configurable parameter,
+# but the default value is probably fine.
+#dynamic_proxy_golang_clean_interval = 10
+
+# The golang proxy needs to know how to talk to your docker daemon. Currently
```

Comment:
This is a point I'm quite worried about but would like to work through (mentally at least) with everyone.

I'd really like to see centralised configuration, if we could define a single .yaml file which defines mappings like

```yaml
---
docker_hosts:
    local:
        host: unix:///...
    r_server:
        host: 192.168.0.3
    j_server:
        host: 192.168.0.2
        TLScert: /path/to/cert.crt

image_distribution:
    rstudio: r_server
    jupyter: j_server
    bam_iobio: any
    my_other_img: local
```

Then I could add support for this in gie-proxy.